src/main/webapp/js/omniComment.js

```diff
@@ -7,9 +7,10 @@ $(document).ready(function() {
     }
     
     function getRedirectSpecificCommentRow() {
-        var start = $(location).attr('href').indexOf('#');
-        var end = $(location).attr('href').length;
-        var rowId = $(location).attr('href').substring(start, end);
+        var url = $(location).attr('href');
```

Comment:
This can simply be ```location.href```, location is a JavaScript object that already has the href property in and of itself, wrapping it as a jQuery object and accessing by ```.attr``` is redundant